torch/_inductor/kernel/flex_attention.py

```diff
@@ -1545,6 +1655,16 @@ def flex_attention(
             mutated_inputs=[
                 logsumexp,
             ],
+            workspace_arg=get_tma_workspace_arg(
+                num_tma_descriptors=3,
+                device=query.get_device(),
+                num_programs=(
+                    (num_heads + cur_kernel_options["BLOCK_M"] - 1)
```

Comment:
can we just call into: https://github.com/pytorch/pytorch/blob/4e1d4333f7a6c9335f1f6d27cc1043b0428c0e42/torch/_inductor/kernel/flex_attention.py#L100

so that they there isn't any future delta

Reply:
updated